src/main/java/com/synopsys/integration/detect/configuration/DetectConfigurationFactory.kt

```diff
@@ -115,6 +116,17 @@ open class DetectConfigurationFactory(private val detectConfiguration: PropertyC
         return null
     }
 
+    @Nullable
+    fun findIndividualFileMatching(): IndividualFileMatching? {
+        val individualFileMatching = detectConfiguration.getValue(DetectProperties.DETECT_BLACKDUCK_SIGNATURE_SCANNER_INDIVIDUAL_FILE_MATCHING)
+
+        if (individualFileMatching.baseValue.isPresent) {
+            return individualFileMatching.baseValue.get()
+        }
```

Comment:
With Optional you can do the following.
`return individualFileMatching.baseValue.orElse(null)`
So it will return the value of baseValue if it is present, or null if it isn't.

Reply:
I'd also argue you may not need a seperate method for this.